src/odemis/driver/test/smartpod_test.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+'''
+Created on Apr 17, 2019
+
+@author: Anders Muskens
+Copyright © 2019 Anders Muskens, Delmic
+
+This file is part of Odemis.
+
+Odemis is free software: you can redistribute it and/or modify it under the terms
+of the GNU General Public License version 2 as published by the Free Software
+Foundation.
+
+Odemis is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+Odemis. If not, see http://www.gnu.org/licenses/.
+'''
+import logging
+import os
+import time
+import unittest
+from odemis.driver import smartpod
+from odemis.util import test
+
+logging.getLogger().setLevel(logging.DEBUG)
+logging.basicConfig(format="%(asctime)s  %(levelname)-7s %(module)s:%(lineno)d %(message)s")
+
+# Export TEST_NOHW=1 to force using only the simulator and skipping test cases
+# needing real hardware
+TEST_NOHW = (os.environ.get("TEST_NOHW", 0) != 0)  # Default to Hw testing
+
+COMP_ARGS = {
+    "atol": 1e-3,
+    "rtol": 1e-3,
+    }
+
+CONFIG = {"name": "SmartPod",
+        "role": "",
+        "locator": "usb:ix:0",
+        # "locator": "fake",
+        "options":"",
+        "axes": {
+            'x': {
+                'range': [-1, 1],
```

Comment:
Could you put more realistic ranges? Based on the hardware we have, a couple of cm would sound more reasonable...